### PR TITLE
chore(TS): complete type check

### PR DESCRIPTION
**Pull request by asturur**

### CODE REVIEW

**Review by ShaMan123 — COMMENTED** [review]
good enough I think

**Comment by ShaMan123** [comment]
coverage dropped

**Comment by ShaMan123** [comment]
The remaining 4 `@ts-nocheck` files are dead files or unmigrated ones

**Review by ShaMan123 — APPROVED** [review]

File: src/shapes/Text/TextSVGExportMixin.ts [hunk 1]
```diff
@@ -293,7 +293,7 @@ export class TextSVGExportMixin extends FabricObjectSVGExportMixin {
    * @return {String}
    */
   getSvgStyles(this: TextSVGExportMixin & Text, skipShadow?: boolean) {
-    // @ts-ignore this line is not error free. ( super is not the same type as this )
+    // @ts-expect-error TS doesn't respect this type casting
```
Comment (asturur) [inline src/shapes/Text/TextSVGExportMixin.ts:296]: to my vscode thise line wasn't an error and LINT was complaining i was using an unused expect-error. Don't you have that?

Comment (asturur) [reply src/shapes/Text/TextSVGExportMixin.ts:296]: here it is in current master 
image

Comment (ShaMan123) [reply src/shapes/Text/TextSVGExportMixin.ts:296]: Same
But the rollup ts plugin throws

**Pull request merged**